imec-int/kepler.gl
# [PREC-105] Add graph layer

## Changes :man_mechanic:

- Added processor to ingest JSON Graph Format https://jsongraphformat.info/
- Added graph layer to show JSON Graph Format

## How do I test this? :hammer:

- The demo app should automatically add a graph layer. Test this one. You can try removing and adding the layer again.

## Image :sunrise_over_mountains:

<img width="932" alt="Screenshot 2022-08-09 at 14 15 27" src="https://user-images.githubusercontent.com/22447514/183644620-441c92bf-82d5-490b-acdb-3ff84174d15d.png">


## Checklist :heavy_check_mark:

- [ ] I have updated the documentation according to my changes.
- [x] I have added the Jira card reference as a prefix to the title of the PR.
